### ISSUE TITLE
Remove parsing of assertion texts

### ISSUE THREAD

**Issue by ahus1** [jboss/jboss-test-audit#12]
**TL;DR:** The `CoverageReport` will replace for example `_` with `<span class="highlight">` which seems do more harm than good in current AsciiDoc/DocBook based workflows. Therefore it should be removed.

The longer version: 

The places where this toolset is being using for example in the Jakarta EE Validation Spec or other specifications like the Sparkplug specification, the `tck-audit.xml` is derived from a DocBook XML file via an XSLT. The DocBook is created from AsciiDoc sources.

The class `CoverageReport` will replace `_`, `~` and `|` with bold, strikethrough and literal HTML elements. 

One way might be to equip the usual XSLT transformations to escape those characters that shouldn't be replaced. On the other hand this would make the XML and all files created from it less readable. 

With the AsciiDoc -> DocBook -> XSLT workflow, I also don't see a way how one would place the special characters to be used in the parsing/replacement into the XML file, as any proper formatting in AsciiDoc would have been translated to DocBook in a way that it is no longer visible to the XSLT step.

Therefore I suggest to remove that parsing. If that can't be achieved in a near future, I'd like to see it deprecated, and with an option to disable that replacement for those who don't want the replacement in the current version.

---

See below for an example in the Sparkplug specification:

Original AsciiDoc source: `[tck-testable tck-id-topic-structure-namespace-unique-edge-node-descriptor]#[yellow-background]*The
group_id combined with the _edge_node_id_ element MUST be unique from any other
group_id/edge_node_id assigned in the MQTT infrastructure.*#`

Extracted in the tck-audit: `The group_id combined with the edge_node_id element MUST be unique from any other group_id/edge_node_id assigned in the MQTT infrastructure.`

Unwanted yellow highlighting in the coverage report: 

![image](https://user-images.githubusercontent.com/3957921/183630636-12218fb5-6c8c-4f89-9c47-33ee8ef64b44.png)

**Comment by manovotn** [comment]
> Therefore I suggest to remove that parsing. If that can't be achieved in a near future, I'd like to see it deprecated, and with an option to disable that replacement for those who don't want the replacement in the current version.

This project isn't actively developed (at least not at the moment), the only recent change was to make it jakarta compatible.
If you can provide a fix, we could figure out a release though.
